cv32e40s/tests/programs/custom/pma/pma.c

```diff
@@ -19,62 +19,81 @@
 #include <stdlib.h>
 #include <stdint.h>
 
-#define  EXCEPTION_INSN_ACCESS_FAULT  1
-#define  EXCEPTION_LOAD_ACCESS_FAULT  5
+#define  EXCEPTION_INSN_ACCESS_FAULT      1
+#define  EXCEPTION_LOAD_ACCESS_FAULT      5
 #define  EXCEPTION_STOREAMO_ACCESS_FAULT  7
-#define  MEM_ADDR_0  0
-#define  IO_ADDR  (0x1A110800 + 16)
-#define  MEM_ADDR_1  0x1A111000
-#define  MTVAL_READ  0
-#define  MTBLJALVEC  0  // TODO update when RTL is implemented
+
+/* CFG Memory Layout
+ *
+ * [r0, "main"]
+ * 0xFFFF_FFFF
+ * 0x1A11_1000         <MEM_ADDR_1>
+ *
+ * [not a region]
+ * 0x1A11_1000
+ * 0x1A11_0800 + 16    <IO_ADDR>
+ *
+ * [r1, "main"]
+ * 0x1A11_0800 + 16
+ * 0x0000_0000         <MEM_ADDR_0>
+ */
+#define  MEM_ADDR_0        0
+#define  IO_ADDR           (0x1A110800 + 16)
+#define  MEM_ADDR_1        0x1A111000
```

Comment:
I believe this is the debug exception handler address - is this deliberate?

Reply:
Yes. This test does not use any debug features, so instead of doing anything difficult with the linker scripts it just re-uses the unused debug memory.